tests/unit_tests/vecpar/rk_stepper_vecpar.cpp

```diff
@@ -0,0 +1,111 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#include "rk_stepper_vecpar.hpp"
+
+#include <gtest/gtest.h>
+
+#include <vecmem/memory/cuda/managed_memory_resource.hpp>
+#include <vecmem/memory/host_memory_resource.hpp>
+
+#include "vecpar/all/main.hpp"
+
+vecpar::config getConfig() {
+
+#if !defined(__CUDA__)
+    vecpar::config config{};  // let the OpenMP runtime choose
+#else
+    constexpr int thread_dim = 2 * 32;
```

Comment:
Would that need manual changing for different devices?

Reply:
Only if you want a specific configuration. If the vecpar functions are called without specifying the setup, the library would choose based on the device type automatically. I set it up here just to have the same grid layout as the CUDA test has.